src/components/views/StoreView.ts

```diff
@@ -56,7 +56,19 @@ export default class StoreView implements ViewComponent {
         const priceSlider = this.createPriceDualSlider();
         const stockSlider = this.createStockDualSlider();
         const copyLinkBtn = this.createCopyLinkBtn();
+        const openCloseBtn = createElem(HTMLTags.BUTTON, 'side-panel-toggle-btn', 'Show filters');
```

Comment:
As far as I can see, you use 'Show filters' twice. Wouldn't it be better to replace it with constant?

Reply:
I think Nina of course fix.
I will merge to develop to update deploy task